worker.go

```diff
@@ -270,15 +320,17 @@ func updateIngress(kubeAdapter *kubernetes.Adapter, item *managedItem) {
 		return
 	}
 	dnsName := strings.ToLower(item.stack.DNSName()) // lower case to satisfy Kubernetes reqs
-	for _, ing := range item.ingresses {
-		if err := kubeAdapter.UpdateIngressLoadBalancer(ing, dnsName); err != nil {
-			if err != kubernetes.ErrUpdateNotNeeded {
-				log.Println(err)
+	for _, ingresses := range item.ingresses {
+		for _, ing := range ingresses {
+			if err := kubeAdapter.UpdateIngressLoadBalancer(ing, dnsName); err != nil {
+				if err != kubernetes.ErrUpdateNotNeeded {
+					log.Println(err)
+				} else {
+					log.Printf("updated ingress not needed %v with DNS name %q", ing, dnsName)
```

Comment:
please switch the error handling and change the message.

    if err == kubernetes.ErrUpdateNotNeeded {
        log.Printf("Ingress update not needed %v with DNS name %q", ing, dnsName)
    } else {
        log.Printf("Failed to update ingress: %v", err)
    }

Reply:
Fixed!